src/osdag/cad/common_logic.py

```diff
@@ -2479,7 +2497,4 @@ def create2Dcad(self):
 
 # if __name__!= "__main__":
 #
-#     CommonDesignLogic()
-
-
-
+#     CommonDesignLogic()
```

Comment:
Consider https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline.

Reply:
Okay, I'll keep it in mind from next time.

Reply:
Could you fix this now as you rebase?